src/jobs.rs

```diff
@@ -359,6 +361,12 @@ impl CommitJobInstance {
         }
     }
 
+    fn get_main_build_ref_name (build_refs: &Vec<models::BuildRef>) -> &str {
```

Comment:
A getter function shouldn't have a get prefix

Reply:
It's not really a getting, I gues. It's more of a helper function to find the main ref. Perhaps calling it `find_main_build_ref_name` would make it clearer?